messaging/src/main/java/org/axonframework/commandhandling/distributed/ReplyMessage.java

```diff
@@ -92,13 +94,21 @@ public ReplyMessage(String commandIdentifier, CommandResultMessage<?> commandRes
 
         if (exceptionDescription != null) {
             return new GenericCommandResultMessage<>(new CommandExecutionException("The remote handler threw an exception",
-                                                                                   new RemoteHandlingException(exceptionDescription),
+                                                                                   convertToRemoteException(exceptionDescription),
                                                                                    payload),
                                                      metaData);
         }
         return new GenericCommandResultMessage<>(payload, metaData);
     }
 
+    private AxonException convertToRemoteException(RemoteExceptionDescription exceptionDescription) {
+        if(exceptionDescription.isPersistent()) {
```

Comment:
Spacing is slightly off here.

Reply:
I am a massive tertiary operator fan, but I feel personal style is also important. 
Anyhow, just wanted to call it out. Feel free to keep it as is or to change it.